src/pages/catalogPage/catalogPage.tsx

```diff
@@ -35,7 +38,7 @@ export function CatalogPage() {
       </div>
       <div className={classes.catalog}>
         <div className={classes.filters}>
-          <CatalogFilters />
+          <CatalogFilters options={categories.categories} />
```

Comment:
Do you use categories outside of CatalogFilters component? I think it might be a good idea to move this logic inside CatalogFilters component